gemrb/core/Strings/String.h

```diff
@@ -31,15 +31,15 @@
 #include "Format.h"
 #include <fmt/xchar.h>
 
-#define WHITESPACE_STRING_W L"\n\t\r "
+#define WHITESPACE_STRING_W u"\n\t\r "
```

Comment:
I'm debating on wether we should change the `W` suffix now. I guess its clear enough it doesn't matter assuming we no longer use `wchar_t` anywhere.

Reply:
Sure, why not.

Reply:
Wait, please check the nature of `WHITESPACE_STRING_VIEW` three lines below.